graylog2-web-interface/src/stores/streams/StreamRulesStore.ts

```diff
@@ -75,6 +76,9 @@ class StreamRulesStore {
   _emitChange() {
     this.callbacks.forEach((callback) => callback());
   }
+  unregister(callback: Callback) {
+    lodash.pull(this.callbacks, callback);
```

Comment:
Using `filter` it would be `this.callbacks = this.callbacks.filter((c) => c !== callback)` which is harder to understand. As lodash also offers a lot of other useful functions that are missing in the javascript standard library, I think it's worth the (maximum) 24kb of overhead.